SubEthaEdit-Mac/Source/PlainTextDocument.m

```diff
@@ -2680,7 +2693,14 @@ - (BOOL)readSEETextFromURL:(NSURL *)anURL properties:(NSDictionary *)aProperties
     [[self session] addContributors:contributors];
     // load text into dictionary
     NSMutableDictionary *storageRep = [dictRep objectForKey:@"TextStorage"];
-    NSString *string = [NSString stringWithContentsOfURL:[NSURL fileURLWithPath:[[anURL path] stringByAppendingPathComponent:@"plain.txt"]]  encoding:[dictRep objectForKey:@"AutosaveInformation"] ? NSUTF8StringEncoding : (NSStringEncoding)[[storageRep objectForKey:@"Encoding"] unsignedIntValue] error:outError];
+    
+    NSURL *textDataURL = [NSURL fileURLWithPath:[[anURL path] stringByAppendingPathComponent:@"plain.txt"]];
```

Comment:
style: on rearranging please also switch to property accessors e.g. `anURL.path` in this case